### DISCUSSION TITLE
Troubleshooting Unsuccessful Folder Deletion in File Session

### DISCUSSION THREAD

**Discussion by amoljagdalepucsd** [Ylianst/MeshCentral#5874] [Issues] [answered]
Hi community, 
In the files session, some folders (both empty and non-empty) are not being deleted, and it's throwing exception
[**fs.unlinkSync(): Error trying to unlink: C:/Users/Default/Videos**] in meshcore.js. I've tried using recursive deletion as well. Could this be a file permissions issue or a problem with the library? Upon inspecting the code, I noticed that we're making the call from default.handlebars and executing the action in meshcore.js file within the deleteFolderRecursive function. In that function, we're simply unlinking the file/folder with fs.unlinkSync(curPath).

**Comment by si458** [comment, marked as answer]
You can't delete the Default folder under Users in windows or anything under it, it's a Windows restriction not a meshcentral issue
